src/neovim.ts

```diff
@@ -116,3 +117,24 @@ export async function downloadStableNeovim(os: Os, token: string | null = null):
         throw err;
     }
 }
+
+export async function buildNeovim(version: string, os: Os): Promise<Installed> {
+    assert.equal(version, 'nightly');
+    assert.equal(os, 'linux');
+    const installDir = path.join(homedir(), 'nvim');
+    core.debug(`Building and installing Neovim to ${installDir}.`);
+    const dir = path.join(await makeTmpdir(), 'neovim');
+
+    await exec('git', ['clone', '--depth=1', 'https://github.com/neovim/neovim', dir]);
+
+    const opts = { cwd: dir };
+    await exec('make', ['-j', `CMAKE_EXTRA_FLAGS=-DCMAKE_INSTALL_PREFIX=${installDir}`], opts);
```

Comment:
```suggestion
    await exec('make', ['-j', `CMAKE_EXTRA_FLAGS=-DCMAKE_INSTALL_PREFIX=${installDir}`, 'CMAKE_BUILD_TYPE=RelWithDebug'], opts);
```

Please add `CMAKE_BUILD_TYPE`. Otherwise it builds Neovim with debug mode.